src/main/resources/.env.example

```
@@ -23,4 +23,45 @@ LOG_LEVEL_SPRING=WARN              # 필요한 경고만 로깅
+
+# oauth
+OAUTH2_GOOGLE_CLIENT_ID=clientId
+OAUTH2_GOOGLE_CLIENT_SECRET=secret
+OAUTH2_GOOGLE_REDIRECT_URI=redirect_uri
+OAUTH2_GOOGLE_GRANT_TYPE=grant_type
+OAUTH2_GOOGLE_SCOPE=scope
+
+OAUTH2_NAVER_CLIENT_ID=D0rAmUvq0gBr3QT8J834
+OAUTH2_NAVER_CLIENT_SECRET=secret
+OAUTH2_NAVER_REDIRECT_URI=redirect_uri
+OAUTH2_NAVER_GRANT_TYPE=grant_type
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The OAUTH2_NAVER_GRANT_TYPE key should go before the OAUTH2_NAVER_REDIRECT_URI key

(UnorderedKey)